plugins/inventory/ovirt.py

```diff
@@ -39,37 +41,42 @@
         env:
           - name: OVIRT_PASSWORD
       ovirt_cafile:
-        description: path to ovirt-engine CA file.
+        description: path to ovirt-engine CA file. If C(ovirt_cafile) parameter is not set and C(ovirt_insecure) is not True, system wide CA certificate store\
+        is used.
         required: False
+      ovirt_insecure:
+        description: A boolean flag that indicates if the server TLS certificate and host name should be checked.
+        required: False
+        default: False
```

Comment:
Isn't default value None?

Reply:
Connection expects a boolean for `insecure`.
As discussed in the original PR (https://github.com/ansible/ansible/pull/49085#discussion_r379341097), this differs in the way `insecure` is handled in the other modules (e.g. ovirt_auth.py) where the value toggles to True in case insecure is None (default) and no CA is provided. I'd prefer to default to False - and possibly fail in the sdk - in case no CA is given and system CA's cannot verify a server.